extensions/vscode/src/views/homeView.ts

```diff
@@ -570,36 +581,51 @@ export class HomeViewProvider implements WebviewViewProvider, Disposable {
           packageFile = rSection.packageFile;
           packageMgr = rSection.packageManager;
 
-          const apiRequest = api.packages.getRPackages(
-            activeConfiguration.configurationName,
-            activeConfiguration.projectDir,
-          );
+          const apiRequest = api.packages
+            .getRPackages(
+              activeConfiguration.configurationName,
+              activeConfiguration.projectDir,
+            )
+            // proposed fix...
+            .catch((error: AxiosError) => {
```

Comment:
Comment this `.catch` out and you'll see the issue when deleting the `renv.lock` file.